lib/hanami/middleware/body_parser.rb

```diff
@@ -8,15 +9,15 @@ module Middleware
     class BodyParser
       # @since 1.3.0
       # @api private
-      CONTENT_TYPE       = 'CONTENT_TYPE'.freeze
```

Comment:
n.b. these constant assignments stoped being fully aligned so I figured it didn't make sense to try and preserve it